tracer/test/Datadog.Trace.Tests/Propagators/B3MultipleHeadersPropagatorTests.cs

```diff
@@ -172,14 +228,16 @@ public void ExtractAndInject_PreserveOriginalTraceId()
                    .Returns(new[] { "1" });
             var result = B3Propagator.Extract(headers.Object);
 
+            // TODO: 128-bit
```

Comment:
I'm guessing we can delete this TODO now?

Reply:
Done, thank you.